github-profiles/JeremyO-7130/README.md

```diff
@@ -0,0 +1,13 @@
+# Hi! I'm Jeremy 🙋🏻🇪🇨
+
+## A brief description about me.
+
+I am a self-taught student who spent some time in college, passionate about technology and love for learning, especially about **web development** 🧑🏻‍💻. I am fascinated to observe all the *functionality* and *aesthetics* of an application when I have finished it.
+
+**JavaScript** has become my go-to tool for most of my projects (although *Python* is out there watching us 👀). 
+
+Anyway, I'm currently learning **React** ⚛️, I find it a delight to use, so I'm constantly improving myself and working hard to master it as well as other dominant technologies. I want to become a competent *full-stack* developer capable of making incredible products, and I will get there 💪🏼
+
+---
```

Comment:
You are missing the most used aliases in your markdown

Reply:
Already added.